packages/tools/generator-component/generators/app/templates/test/specs/component/f-skeleton.component.__spec__.js

```diff
@@ -1,14 +1,14 @@
-import <%= name.filename %>Component from '../../../test-utils/component-objects/f-<%= name.class %>.component';
+const <%= name.filename %> = require('../../../test-utils/component-objects/f-<%= name.class %>.component');
+const <%= name.class %> = new <%= name.filename %>
 
 describe('f-<%= name.class %> component tests', () => {
     beforeEach(() => {
-        browser.url('?path=<%= storybook.path %>');
-        browser.switchToFrame(0);
-        <%= name.filename %>Component.waitFor<%= name.filename %>Component();
+        <%= name.class %>.open();
```

Comment:
Not a comment to change anything here, but should we update the tests in other components to match this too?

Reply:
yep! Organisms are already updated and I've almost done updating the atoms and molecules too 😄